### ISSUE TITLE
Any plans for intereactivity in the future?

### ISSUE THREAD

**Issue by VarLad** [davibarreira/Vizagrams.jl#56]
A discussion thread would be more appropriate, but I was wondering if there are any plans for integrating some interactivity mechanics in the visualizations, in the future.

**Comment by davibarreira** [comment]
Yes. There are plans for interactivity. One of the ways I'm implementing it is by directly injecting javascript so that one can manipulate the svg generated. The other one is to implement a new backend other than simple SVG.

The implementation with javascript injection has already advanced. I might release a tool-tip capability in some days.
